runner.vm/files/usr/local/bin/runner.sh

```diff
@@ -7,9 +7,11 @@ GITHUB_RUNNER_REG_URL=$(${CURL} "http://metadata.google.internal/computeMetadata
 GITHUB_RUNNER_TOKEN=$(${CURL} "http://metadata.google.internal/computeMetadata/v1/instance/attributes/runnerToken" -H "Metadata-Flavor: Google")
 GITHUB_RUNNER_LABEL=$(${CURL} "http://metadata.google.internal/computeMetadata/v1/instance/attributes/runnerLabel" -H "Metadata-Flavor: Google")
 
-RUNNER_RELEASE_URL=$(${CURL} https://api.github.com/repos/actions/runner/releases | \
-	             jq -r 'first(.[] | select(.prerelease == false)) | .assets[] | if .name | test("actions-runner-linux-x64-[0-9.]+.tar") then .browser_download_url else empty end')
-${CURL} "${RUNNER_RELEASE_URL}" | tar zx
+RUNNER_RELEASE_URL=$(${CURL} https://api.github.com/repos/actions/runner/releases/latest | \
+                     jq -e -r '.assets[] | if .name | test("actions-runner-linux-x64-[0-9.]+.tar") then .browser_download_url else empty end')
+${CURL} -O "${RUNNER_RELEASE_URL}"
```

Comment:
The `jq -e` flag is a new trick for me, nice.

Should we quote these variable assignments to protect against BASH code injection?
```bash
RUNNER_RELEASE_URL=$(...)
```
...would become:
```bash
RUNNER_RELEASE_URL="$(...)"
```

As a point of interest, I believe the `jq` query `if-then` snippet...
```
if .name | test("actions-runner-linux-x64-[0-9.]+.tar") then .browser_download_url else empty end
```
...could be simplified to this.
```
.name | test("actions-runner-linux-x64-[0-9.]+.tar") | .browser_download_url // empty
```
I haven't tried it, not certain.

Reply:
> Should we quote these variable assignments to protect against BASH code injection

Yeah, probably prudent to do so just for best practices purposes.

Not sure if I exactly understand your suggestion. It's easy enough to try -- no token needed or anything. If you find a more concise way I'd certainly be interested.
```
$ curl -s https://api.github.com/repos/actions/runner/releases/latest | jq -e -r '.assets[] | .name | test("actions-runner-linux-x64-[0-9.]+.tar") | .browser_download_url'
jq: error (at <stdin>:1242): Cannot index boolean with string "browser_download_url"
```

Reply:
Nevermind on the `jq` thing, I see the `if...else` block behaves slightly different than I had thought when I wrote that comment. My mistake.